resource_booking/models/resource_booking.py

```diff
@@ -137,6 +172,14 @@ class ResourceBooking(models.Model):
         tracking=True,
         help="Who will attend this booking?",
     )
+    partner_ids = fields.Many2many(
```

Comment:
This one shouldn't be added in this commit.

Reply:
The final commit I added removes this (I didn't want to modify the author's commits).

Reply:
I think this is confusing in this case. Better to not change it. And this should be already existing in 15.0 previous commits.